cmd/up/up.go

```diff
@@ -255,22 +255,37 @@ func (up *upContext) start(autoDeploy, build bool) error {
 // activateLoop activates the development container in a retry loop
 func (up *upContext) activateLoop(autoDeploy, build bool) {
 	isRetry := false
-	t := time.NewTicker(3 * time.Second)
+	isTransientError := false
+	t := time.NewTicker(1 * time.Second)
+	iter := 0
 	defer t.Stop()
 
 	for {
+		if isRetry || isTransientError {
+			log.Infof("waiting for shutdown sequence to finish")
+			<-up.Canceled
+			if iter == 0 {
+				log.Yellow("Connection lost to your development container, reconnecting...")
+			}
+			iter++
+			iter = iter % 10
+			if isTransientError {
```

Comment:
can we also add the log.Yellow here? or maybe use the same animation pattern that we do for the up? I'd like to make it visible that we're trying to reconnect rather than being stuck.

@rlamana what do you think?

Reply:
@rberrelleza the message is shown every 10s